src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -3764,7 +3809,33 @@ public void mousePressed(MouseEvent e) {
             + "\n be used by PC^2 to display additional information to the Judges and/or Teams."
 
             + "\n\nFor more information, see the PC^2 Contest Administrator's Guide; in particular, the Appendix on Validators."
-            + "\nSee also the CLICS specification at https://clics.ecs.baylor.edu/index.php/Problem_format#Validators.";
+            + "\nSee also the CLICS specification at https://icpc.io/problem-package-format/spec/problem_package_format#output-validators.";
```

Comment:
I agree.  It is horrendous that all these huge string constants for HELP messages are in actual source code.  These should be moved to a separate resource or file and looked up as needed.  This way they could easily be modified without a code change (when things like URL's change!).   But, out of scope here.  This can be part of the refactor plan.